Analyzer/plugins/Analyzer.cc

```diff
@@ -2334,17 +2397,20 @@ void Analyzer::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup)
           // 0.31623 [Bichsel's smallest entry]  && genGammaBeta > 0.31623
           if (!isData && (globalIas_ > 0.6 || (globalIas_ > 0.02 && globalIas_ < 0.03))) {
             if (!headerPixPrintedAlready) {
-              std::cout << std::endl << "        | $I_{as}$ | Layer | gammaBeta | flipped | cotAlpha | cotBeta | momentum | sizeX | sizeY";
-              std::cout << " | Norm. Charge | edge | bad | double | cProbXY | cProbQ | " << std::endl;
-              std::cout << "        |--- | ---| " << std::endl;
-              
+              // std::cout << std::endl << "        | $I_{as}$ | Layer | gammaBeta | flipped | cotAlpha | cotBeta | momentum | sizeX | sizeY";
+              // std::cout << " | Norm. Charge | edge | bad | double | cProbXY | cProbQ | " << std::endl;
+              // std::cout << "        |--- | ---| " << std::endl;
```

Comment:
well these are actually very useful things to get printed... Can you add them back under a debug > 0 condition?